README.md

```diff
@@ -30,6 +30,33 @@ There are plenty of small, self-contained examples [in the documentation][exampl
 
 The easiest way to get started using ct is to download one of the binaries from the [releases page on GitHub][releases].
 
+One can use the following script to download and verify the signature of Config Transpiler:
+
+```bash
+# Sepcify Config Transpiler version
+CT_VER=v0.6.1
+# Sepcify OS
+OS=apple-darwin # MacOS
+OS=unknown-linux-gnu # Linux
```

Comment:
For users copy/pasting/running this script, setting one OS and then immediately overriding it with another is redundant. Perhaps default to `OS=unknown-linux-gnu`, and having commented out lines above it for MacOS and windows?